packages/seal-query-translator/composer.json

```diff
@@ -0,0 +1,78 @@
+{
+    "name": "cmsig/seal-query-translator",
```

Comment:
not sure about the name yet it generate / create filters out of a Galach query language using netgen/query-translator package.